src/components/Calculator.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
```

Comment:
- [ ] Kindly be noticed that the `Calculator` component should import the `calculate.js` file from the logic directory.
```javascript
import React, { Component } from 'react';
import calculate from ../logic;
```